src/components/Header/DrawerCart.tsx

```diff
@@ -111,7 +111,12 @@ export default function DrawerCart(props: Props) {
             >
               {t("cartDetail.description")}
             </Typography>
-            <Typography component="div" typography="sidenav" display="block">
+            <Typography
+              component="div"
+              typography="sidenav"
+              display="block"
+              style={{ overflowWrap: "break-word" }}
```

Comment:
What do you think about setting this property in the upper div? In this way if the other fields are too long they are also handled.

Reply:
The only fields that can cause this problem are {el.description} and {el.companyName}, since the other fields are fixed lenght fields (Notice code and Payee tax code), while the price cant ever reach the critical number of digits.

Reply:
Ok perfect